src/Hazelcast.Net/Models/CacheSimpleEntryListenerOptions.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Hazelcast.Configuration;
+using Hazelcast.Serialization;
+using System;
+
+namespace Hazelcast.Models;
+
+public class CacheSimpleEntryListenerOptions : IIdentifiedDataSerializable
```

Comment:
Is this option used somewhere or related with map or ringbuffer?

Reply:
It is indeed not used anywhere. Is it a leftover? In any case it should then be `internal`. Stephan to review all of them and mark `internal` where appropriate.

Reply:
Fixed with e23ed07ca it seems to be the only one